builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -317,22 +314,22 @@ function store.update_paths()
 	pkgmgr.refresh_globals()
 	for _, mod in pairs(pkgmgr.global_mods:get_list()) do
 		if mod.author then
-			mod_hash[mod.author .. "/" .. mod.name] = mod
+			mod_hash[mod.author:lower() .. "/" .. mod.name] = mod
 		end
 	end
 
 	local game_hash = {}
 	pkgmgr.update_gamelist()
 	for _, game in pairs(pkgmgr.games) do
-		if game.author then
-			game_hash[game.author .. "/" .. game.id] = game
+		if game.author ~= "" then
```

Comment:
I don't think this should work. `author` should either be filled or nil, but never an empty string

Reply:
Actually the tables returned by `get_games` always have the author field
https://github.com/minetest/minetest/blob/master/src/script/lua_api/l_mainmenu.cpp#L434